.circleci/config.yml

```diff
@@ -195,3 +198,24 @@ workflows:
       - comment:
           requires:
             - deploy
+  approval:
+    jobs:
+      - ready:
+          filters:
+            branches:
+              ignore: main
+          type: approval
```

Comment:
I don't think we should necessarily block on this... Also doesn't the chromatic GitHub app handle the approval part? Why would we need circle to do that?

Reply:
I think you're reading this a little differently
The flow here is that for the chromatic test to run, someone must approve that the chromatic test can be run

This means that chromatic isn't run on every commit, which would be costly, and is only run once we are ready to run it because we're thinking about merging.

The `main-workflow` below takes care of running chromatic once something is merged into main, during this workflow, it auto accepts everything, this is to make sure that main is clean.

There is definitely room for improvement on the ready job that requires approval, for instance, we could put a job in that flow that determines if it's just docs and skips the rest if that's the case. Unfortunately optional approval jobs are not a thing right now https://discuss.circleci.com/t/workflows-with-optional-approval-jobs-dont-appear-as-successful-in-github-or-circleci/31292

Let me know if i missed the mark and you had something else in mind